opentelemetry-sdk/benches/noop_metrics.rs

```diff
@@ -0,0 +1,85 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use opentelemetry::{
+    metrics::{noop::NoopMeterProvider, Counter, MeterProvider as _},
+    KeyValue,
+};
+
+use rand::{rngs::SmallRng, Rng, SeedableRng};
+
+fn create_counter() -> Counter<u64> {
+    let meter_provider: NoopMeterProvider = NoopMeterProvider::default();
+    let meter = meter_provider.meter("benchmarks");
+    let counter = meter.u64_counter("counter_bench").init();
+    counter
+}
+
+fn criterion_benchmark(c: &mut Criterion) {
+    noop_counter_add(c);
+    create_keyvalue_vector(c);
+}
+
+fn noop_counter_add(c: &mut Criterion) {
+    let attribute_values = [
+        "value1", "value2", "value3", "value4", "value5", "value6", "value7", "value8", "value9",
+        "value10",
+    ];
+
+    let mut rng = SmallRng::from_entropy();
```

Comment:
since we are measuring noops only, there is no need of random attributes.

Reply:
Yes, kept the code as it is not costed as part of the benchmark. Will make it static array.